### ISSUE TITLE
Cannot resolve package

### ISSUE THREAD

**Issue by stratospark** [duckdb/duckdb-wasm#530]
Our custom bundler script couldn't handle this package. After some investigation I discovered a typo in the `package.json` which I've fixed in #529 

Current Behavior:
```
PS C:\Users\patr\Code\yarn-test> node
Welcome to Node.js v16.13.1.
Type ".help" for more information.
> require.resolve('./node_modules/@duckdb/duckdb-wasm');
Uncaught Error: Cannot find module './node_modules/@duckdb/duckdb-wasm'
Require stack:
- <repl>
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.resolve (node:internal/modules/cjs/helpers:108:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ '<repl>' ]
```

Expected Behavior:
```
> require.resolve('./node_modules/@duckdb/duckdb-wasm');
'C:\\Users\\patr\\Code\\yarn-test\\node_modules\\@duckdb\\duckdb-wasm\\dist\\duckdb-browser.cjs'
```

**Comment by stratospark** [comment]
@ankoh When can we expect a new build to be pushed to npm? I'm looking forward to integrating this at work 😄

**Comment by ankoh** [comment]
Awesome! Published in v1.13.0.
